engines/mohawk/livingbooks_code.cpp

```diff
@@ -1957,7 +1957,7 @@ uint LBCode::parseCode(const Common::String &source) {
 				Common::String tempString;
 				tempString += token;
 				while (pos < source.size()) {
-					if (!Common::isAlpha(source[pos]) && !Common::isDigit(source[pos]))
+					if (!Common::isAlpha(source[pos]) && !Common::isDigit(source[pos]) && source[pos] != 95)
```

Comment:
This magic value will need a comment to explain why it has been added. Will that break any of the other supported living books games?

Reply:
@bluegr : As per my comment, this is explained in the PR header text, but not in the code
````
The && source[pos] != 95 part is due to some variables using _ in the names, which the engine was previously failing on parsing.
````

95 Decimal is ASCII / UTF-8 for the underscore character. As you said, this should be commented, or it can be replaced by the following for self-documentation:
````
&& source[pos] != '_'
````

Reply:
Please add it to the code as well :) And switch to a character constant. E.g. do both

Reply:
Done